### PR TITLE
SDI-265: ♻️  Convert contact-us to nunjucks

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: server/middleware/setUpAuthentication.ts [hunk 1]
```diff
@@ -44,6 +45,10 @@ export default function setUpAuth(userAuthenticationService: UserAuthenticationS
     } else res.redirect(authLogoutUrl)
   })
 
+  router.use('/account-details', (req, res) => {
```
Comment (petergphillips) [inline server/middleware/setUpAuthentication.ts:48]: use same approach as typescript template here

File: server/routes/index.ts [hunk 2]
```diff
@@ -14,11 +14,7 @@ export default function indexRouter(router: Router, services: Services): Router
 
   get('/', (req, res) => res.redirect('/calendar#today'))
   get('/contact-us', (req, res) => {
-    res.render('pages/contact-us', {
-      employeeName: req.user?.employeeName,
```
Comment (petergphillips) [inline server/routes/index.ts:18]: this is now in `res.locals` so doesn't need to be exposed manually here anymore

File: server/routes/index.ts [hunk 3]
```diff
@@ -14,11 +14,7 @@ export default function indexRouter(router: Router, services: Services): Router
...
-      csrfToken: res.locals.csrfToken,
```
Comment (petergphillips) [inline server/routes/index.ts:19]: `csrfToken` was already in `res.locals` so available in nunjucks templates

File: server/routes/index.ts [hunk 4]
```diff
@@ -14,11 +14,7 @@ export default function indexRouter(router: Router, services: Services): Router
...
-      authUrl: req.authUrl,
```
Comment (petergphillips) [inline server/routes/index.ts:20]: we've now got a redirect from check my diary so don't need this passing through either

File: server/routes/testutils/appSetup.ts [hunk 5]
```diff
@@ -21,9 +21,11 @@ function appSetup(route: Router, production: boolean, hmppsAuthMFAUser: boolean)
     req.user = {
       employeeName: 'first last',
       authSource: 'nomis',
-      token: 'token',
+      token:
+        'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6ImZpcnN0IGxhc3QiLCJpYXQiOjE1MTYyMzkwMjIsImF1dGhvcml0aWVzIjoiUk9MRV9NRkEifQ.-oYWVIbTyzjcrd9XqG5zyh5nf9G_zcLrXy2BxCphexM',
```
Comment (petergphillips) [inline server/routes/testutils/appSetup.ts:25]: sample token generated from https://jwt.io/ that just contains `name` and `authorities`

File: server/views/pages/contact-us.njk [hunk 6]
```diff
@@ -0,0 +1,25 @@
+{% extends "../partials/layout.njk" %}
+
+{% set pageTitle = applicationName + " - Contact us" %}
+
+{% block beforeContent %}
+  <a href="/" class="govuk-back-link">Back</a>
+{% endblock %}
+
+{% block content %}
+  <main id="main-content" role="main">
```
Comment (petergphillips) [inline server/views/pages/contact-us.njk:10]: content copied from ejs version

File: server/views/partials/header.njk [hunk 7]
```diff
@@ -3,15 +3,15 @@
     <div class="dps-header__title float-left govuk-!-padding-top-2">
       {% include "./crest-title.njk" %}
     </div>
-    {% if employeeName %}
+    {% if user %}
```
Comment (petergphillips) [inline server/views/partials/header.njk:6]: same approach as in typescript template

**Review by steverendell — DISMISSED** [review]
lgtm

File: server/routes/testutils/appSetup.ts (hunk 5 above)
Comment (steverendell) [reply server/routes/testutils/appSetup.ts:25]: There is a security scan of github which might pick this up (even though it is a false positive) . Could the token be generated dynamically?

File: package.json [hunk 8]
```diff
@@ -111,7 +111,6 @@
     "helmet": "^5.1.0",
     "http-errors": "^2.0.0",
     "ip-range-check": "^0.2.0",
-    "jsonwebtoken": "^8.5.1",
```
Comment (petergphillips) [inline package.json:114]: this only needed to be a dev dependency anyway

**Review by steverendell — APPROVED** [review]
lgtm

**Pull request merged**